src/web/configuration.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  UserConfiguration,
+  ExtendedConfiguration,
+  setDefaults as setBaseDefaults,
+} from '../core/interfaces/configuration';
+
+// --------------------------------------------------------
+// ----------------------- Defaults -----------------------
+// --------------------------------------------------------
+// region Defaults
+
+/**
+ * Whether PubNub client should update its state using browser's reachability events or not.
+ *
+ * If the browser fails to detect the network changes from Wi-Fi to LAN and vice versa, or you get
+ * reconnection issues, set the flag to `false`. This allows the SDK reconnection logic to take over.
+ */
+const LISTEN_TO_BROWSER_NETWORK_EVENTS = true;
+
+/**
+ * Whether PubNub client should try utilize existing TCP connection for new requests or not.
+ */
+const KEEP_ALIVE = true;
```

Comment:
Great! to keep it true default.
also, assuming that it wont have any side effect.

Reply:
It looks like `true` is the default for `fetch` Web API itself in web environment.